packages/mobile/src/components/AvatarSelf.tsx

```diff
@@ -36,5 +36,12 @@ export function AvatarSelf({ iconSize, displayNameStyle }: Props) {
     }
   }
 
-  return <Avatar recipient={recipient} iconSize={iconSize} displayNameStyle={displayNameStyle} />
+  return (
```

Comment:
Why does this file need to exist? From what I can see,`Avatar` only ever refers to the active user. Seems like we don't need both of these.

Reply:
It's used by recipients on `ReclaimPaymentConfirmationCard`, but it might not be necessary anyways, I'll check

Reply:
`Avatar` has two uses:
- `ReclaimPaymentConfirmationCard` -> Has a TODO to use the `recipient` field but just passes the `e164Number`. It's for invited users.
- `QRCode` -> Uses `AvatarSelf` which creates a `recipient` with the user's detail and passes that to `Avatar.`

Reply:
Gotcha. IMO having `Avatar` and `AvatarSelf` is kind of unnecessary if the only point of `AvatarSelf` is to format a `recipient` object with the user's info. Making this change is not strictly in scope for this PR though so up to you if you want to do it